data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -165,16 +177,23 @@ def transform(self) -> None:
 
         # merge census cbgs with score
         merged_df = cbg_usa_df.merge(
-            self.score_county_state_merged, on="GEOID10", how="left"
+            self.score_county_state_merged,
+            on="GEOID10",
+            how="left",
+        )
+
+        # recast population to integer
+        merged_df["Total population"] = (
+            merged_df["Total population"].fillna(0.0).astype(int)
         )
 
         # list the null score cbgs
         null_cbg_df = merged_df[merged_df["Score E (percentile)"].isnull()]
 
         # subsctract data sets
-        removed_df = pd.concat([merged_df, null_cbg_df, null_cbg_df]).drop_duplicates(
-            keep=False
-        )
+        removed_df = pd.concat(
+            [merged_df, null_cbg_df, null_cbg_df]
```

Comment:
Noticed this earlier today, should there by two `null_cbg_df`s?

Reply:
Will leave a comment on the approach in-line.